src/encord_active/lib/file_structure/base.py

```diff
@@ -70,7 +69,7 @@ def project_meta(self) -> Path:
 
     @abstractmethod
     def data_units(
-        self, pattern: Optional[list[DataUnitLike]] = None, include_label_row: bool = False
+        self, where: Optional[list[DataUnitWhereInput]] = None, include_label_row: bool = False
     ) -> Iterator[models.DataUnit]:
```

Comment:
The output should be list as in `ProjectFileStructure`. The same for `BaseProjectFileStructure.label_rows()`.